beacon-chain/rpc/apimiddleware/structs.go

```diff
@@ -53,24 +53,6 @@ type StateFinalityCheckpointResponse_StateFinalityCheckpointJson struct {
 	Finalized         *CheckpointJson `json:"finalized"`
 }
 
-type StateValidatorsResponseJson struct {
-	Data                []*ValidatorContainerJson `json:"data"`
```

Comment:
You can remove `ValidatorContainerJson` from line 986 too since it is no longer in use.